config/templates.js

```diff
@@ -4,8 +4,8 @@ module.exports = {
     repo: 'pages-uswds-11ty',
     branch: 'main',
     engine: 'node.js',
-    example: 'https://federalist-32bdb3ca-a71a-46e5-ab20-a38ecfdbf143.app.cloud.gov/site/cloud-gov/pages-uswds-11ty/',
-    title: '11ty - U.S. Web Design System v2.0',
+    example: 'https://federalist-32bdb3ca-a71a-46e5-ab20-a38ecfdbf143.sites.pages.cloud.gov/site/cloud-gov/pages-uswds-11ty/',
+    title: '11ty - U.S. Web Design System v3.0',
     description: 'A flexible multi-page setup intended to introduce an agency or program. It offers an engaging Hero section for a current priority and a landing page layout designed to provide clarity and context for first time visitors. This template also offers navigation options for internal pages.',
     thumb: '/images/pages-uswds-v2-thumbnail.png',
```

Comment:
Should we rename and/or recreate the thumbnail?

Reply:
Probably. Also, should we create custom domains for these?

Reply:
ie. `uswds-11ty.pages.cloud.gov`

Reply:
Updated. Will wait on https://github.com/cloud-gov/cg-provision/pull/1134 before merging.